packages/hashids/hashids.1.0.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "Vincent Jacques <vincent@vincent-jacques.net>"
+authors: "Vincent Jacques <vincent@vincent-jacques.net>"
+homepage: "https://jacquev6.github.io/hashids-ocaml/"
+bug-reports: "http://github.com/jacquev6/hashids-ocaml/issues/"
+license: "MIT"
+doc: "https://jacquev6.github.io/hashids-ocaml/"
+dev-repo: "https://github.com/jacquev6/hashids-ocaml.git"
+build: ["ocamlbuild" "-use-ocamlfind" "Hashids.cma" "Hashids.cmxa"]
+install: [
+  ["mkdir" "-p" "%{prefix}%/lib/hashids"]
+  [
+    "cp"
+    "META"
+    "_build/Hashids.cmi"
+    "_build/Hashids.cma"
+    "_build/Hashids.a"
+    "_build/Hashids.cmx"
+    "_build/Hashids.cmxa"
+    "%{prefix}%/lib/hashids"
```

Comment:
same thing here: this is very fragile and will fail if you don't have a native compiler installer (not sure what will happen on windows). Not sure if it is a very good to merge this for a 1.0 version, but this is up to you.